crates/ty_python_semantic/src/types/enums.rs

```diff
@@ -0,0 +1,128 @@
+use rustc_hash::FxHashSet;
+
+use crate::{
+    Db,
+    place::{Place, place_from_bindings, place_from_declarations},
+    semantic_index::{place_table, use_def_map},
+    types::{ClassLiteral, KnownClass, Type},
+};
+
+/// List all members of an enum.
+pub(crate) fn enum_members<'db>(db: &'db dyn Db, class: &'db ClassLiteral<'db>) -> Vec<String> {
```

Comment:
This will certainly have a richer return type in the future. For now, just return a list of strings to power the `ty_extensions.enum_members` functionality.